arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json

```diff
@@ -487,18 +487,27 @@
           "type": "string",
           "description": "Required. Indicates the type of cognitive service account.",
           "enum": [
+            "Academic",
+            "Bing.Autosuggest",
+            "Bing.Search",
+            "Bing.Speech",
+            "Bing.SpellCheck",
             "ComputerVision",
+            "ContentModerator",
             "Emotion",
             "Face",
             "LUIS",
             "Recommendations",
+            "SpeakerRecognition",
             "Speech",
+            "SpeechTranslation",
             "TextAnalytics",
+            "TextTranslation",
             "WebLM"
           ],
           "x-ms-enum": {
             "name": "Kind",
-            "modelAsString": false
+            "modelAsString": true
```

Comment:
this is a breaking change in multiple ways
1. An Enum is being converted to a static class with string constants. This affects the signature of the constructor for "CognitiveServicesAccountCreateParameters" model.
2. New values are added haphazardly instead of appending new values to the existing set.